slices/libmysqlclient21.yaml

```diff
@@ -0,0 +1,20 @@
+package: libmysqlclient21
+
+essential:
+  - libmysqlclient21_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - libssl3t64_libs
+      - libstdc++6_libs
+      - libzstd1_libs
+      - mysql-common_config
+      - zlib1g_libs
+    contents: 
+      /usr/lib/*-linux-gnu/libmysqlclient.so.21*:
```

Comment:
Let's match style of other SDFs and use `/usr/lib/*-linux-*/libmysqlclient.so.21*` in referencing the file.